golem_messages/message/tasks.py

```diff
@@ -107,6 +108,9 @@ def __init__(self, header: datastructures.MessageHeader = None,
         if self.concent_enabled is None:
             self.concent_enabled = True
 
+        if isinstance(self.subtask_price, int):
+            assert self.subtask_price > 0
```

Comment:
@kbeker is this requirement specified anywhere in the documentation? @lukasz-glen perhaps?

Reply:
@kbeker ah, just noticed @lukasz-glen 's comment in the issue